finder-app/manual-linux.sh

```diff
@@ -24,6 +25,7 @@ fi
 mkdir -p ${OUTDIR}
 
 cd "$OUTDIR"
+echo "Checkpoint 1"
```

Comment:
I like the idea to add prints, but something more verbose like "Preparing for kernel build" might be better.